node/node_newblock.go

```diff
@@ -78,7 +78,16 @@ func (node *Node) WaitForConsensusReadyV2(readySignal chan struct{}, stopChan ch
 func (node *Node) proposeNewBlock() (*types.Block, error) {
 	// Update worker's current header and state data in preparation to propose/process new transactions
 	coinbase := node.Consensus.SelfAddress
-	node.Worker.UpdateCurrent(coinbase)
+
+	// After staking, all coinbase will be the address of bls pub key
```

Comment:
would be nice if was a longer comment explaining why specifically after staking, the connection between address(bls_pub_key) (I thought address were always ecdsa based) and coin base